cicd/version_info.sql

```diff
@@ -9,7 +9,7 @@ WITH versions(name, version) AS (
     FROM
         pg_available_extensions
     WHERE
-        name IN ('timescaledb', 'postgis', 'pg_prometheus', 'hotforge', 'timescale_prometheus_extra', 'promscale', 'timescaledb_toolkit', 'timescale_analytics', 'timescaledb_cloudutils', 'timescaledb_osm')
```

Comment:
Why are we removing these extensions from the where clause?
(If the image doesn't contain the extension, the resulting rows would not include these anyway).

Reply:
Just to remove internal timescale parts from this public image.